kernel/src/main/java/org/kframework/unparser/WantsPrettyPrinter.java

```diff
@@ -0,0 +1,12 @@
+package org.kframework.unparser;
```

Comment:
copyright on first line

Reply:
I remember there was a check for copyright at build time. Why it was not triggered?